AsyncDisplayKit/Details/ASThread.h

```diff
@@ -29,6 +29,7 @@ static inline BOOL ASDisplayNodeThreadIsMain()
 #ifdef __cplusplus
 
 #define TIME_LOCKER 0
+#define CHECK_LOCKING_SAFETY 0
```

Comment:
This is neat! It would be really cool if you could write up a comment explaining how to use it and how it can be used to find threading issues.